src/components/Docs/SnippetViewer/SupportedLanguage.tsx

```diff
@@ -64,3 +64,5 @@ export interface LanguageMapping {
 export interface LanguageMappings {
   [key: string]: LanguageMapping;
 }
+
+export const DefaultAuthorizationModelId = '1uHxCSuTP0VKPYSnkq1pbb1jeZw';
```

Comment:
Ideally, this would be in another place, but we can merge for now

Reply:
when we have more of these const, we can move it to another file